sources/packages/api/src-sql/Types/Revert-scholastic-standing-status-enum.sql

```diff
@@ -0,0 +1,16 @@
+-- Remove pending and declined from scholastic_standing_status.
```

Comment:
I believe that we can remove the entire type.

Reply:
mm, so are you suggesting to update the below field to a  string 
![image](https://user-images.githubusercontent.com/77353155/171476834-b54ad3de-cc5c-4159-beb6-f2606e2db173.png)

Reply:
yup make sense, removing the status column itself

Reply:
@ann-aot I believe @andrewsignori-aot  is suggesting to remove the column itself scholastic_standing_status which eventually leads to removal of type sims.scholastic_standing_status.